lib/src/platform_client.dart

```diff
@@ -408,10 +399,6 @@ class PlatformClient {
   }
 
   void _initSession() {
-    // Initialize the RabbitMQ client.
-    _mq?.dispose();
-    _mq = PlatformMQImpl(_config.environment, _session!);
```

Comment:
Before, we were connecting to the MQTT broker after logging in and disconnecting when logging out. There were a couple issues with this approach:
1. Since we only need the MQTT connection during service requests, we could be wasting resources.
2. We don't have enough information after logging in to create a meaningful LWT message.